pkg/azclient/fileshareclient/zz_generated_client.go

```diff
@@ -56,13 +56,13 @@ func New(subscriptionID string, credential azcore.TokenCredential, options *arm.
 const GetOperationName = "FileSharesClient.Get"
 
 // Get gets the FileShare
-func (client *Client) Get(ctx context.Context, resourceGroupName string, parentResourceName string, resourceName string) (result *armstorage.FileShare, err error) {
+func (client *Client) Get(ctx context.Context, resourceGroupName string, parentAccountName string, targetFileShareName string) (result *armstorage.FileShare, err error) {
```

Comment:
this is still incorrect, let's use azure file naming in https://github.com/kubernetes-sigs/cloud-provider-azure/pull/7718

Reply:
These files are generated. Please refer to changes in template.go